.github/workflows/build.yml

```diff
@@ -40,6 +40,9 @@ jobs:
     - name: Build OSGi bundle
       run: npm run build-plugin
 
+    - name: prepublish
+      run: npm run vscode:prepublish
```

Comment:
Suggest moving this part to script `npm run test` in package.json, to make it more convenient in development.

Reply:
Make sense

Reply:
get it